packages/story-editor/src/components/elementLink/output.js

```diff
@@ -31,14 +31,17 @@ function WithLink({ element, children, ...rest }) {
   if (!link?.url?.length) {
     return children;
   }
+  const rel = link?.rel.join(' ') ?? '';
```

Comment:
I don't think the null coalescing operator does anything here.

If `rel` is not an array, this will throw an error.

If it's an empty array, it's never null, so the `''` fallback is never hit.

This would be safer:

```suggestion
  const rel = link?.rel?.join(' ') || '';
```